### PR TITLE
Fixed fallback function in the FirstOf class

### PR DESCRIPTION
**Pull request by fanifieiev**
This PR is for issue [1276](https://github.com/yegor256/cactoos/issues/1276)

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#1306](https://github.com/yegor256/cactoos/issues/1306) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C63314D6Z/5e8b0068-6521-429a-ac56-ae764f5bc954); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by paulodamaso** [comment]
@0crat assign me

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/yegor256/cactoos/issues/1306#issuecomment-585721742))

@paulodamaso This pull request [#1306](https://github.com/yegor256/cactoos/issues/1306) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/C63314D6Z/920ada65-1d8a-45c6-96bb-9b01803eaa0a); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

File: src/test/java/org/cactoos/scalar/FirstOfTest.java [hunk 1]
```diff
@@ -79,4 +81,23 @@ public void returnsFallbackIfNothingMatches() {
         );
     }
 
+    @Test
+    public void shouldThrowExceptionForNotFoundElementTest() {
+        new Assertion<>(
+            "Should not return anything",
```
Comment (paulodamaso) [inline src/test/java/org/cactoos/scalar/FirstOfTest.java]: @fanifieiev Th is test looks a little confused to me:
- the method signature says that we are testing if we are throwing an exception for element not found, but the message says does not state about that;
- essentially this test is testing the same thing that `returnsFallbackIfNothingMatches`

Comment (fanifieiev) [reply src/test/java/org/cactoos/scalar/FirstOfTest.java]: @paulodamaso Right, will fix it

**Comment by fanifieiev** [comment]
@paulodamaso Please have a look

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was finished: +15 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Pull request merged**